lib/Immediate/Immediate.cpp

```diff
@@ -341,6 +338,7 @@ int swift::RunImmediately(CompilerInstance &CI,
              Module->dump());
 
   {
+    std::unique_ptr<llvm::LLVMContext> ModuleCtx(&Module->getContext());
     auto TSM = llvm::orc::ThreadSafeModule(std::move(Module), std::move(ModuleCtx));
```

Comment:
This doesn't seem like the right mode of use of this ORC API. It wants total ownership of the module context, and we're handing it effectively a borrowed reference from the module itself. I *think* you can spawn another context here, but @lhames is the authority here.

Reply:
Even worse! The module doesn't own its underlying context. Once the IRGenerator goes Kaput, we have no guarantees this pointer is valid.

Reply:
Thanks. I was wondering about that.